lib/rx/client.rb

```diff
@@ -91,7 +91,10 @@ def raise_not_authenticated
     end
 
     def connection
-      @connection ||= Faraday.new(config.base_path, headers: BASE_REQUEST_HEADERS, request: request_options)
+      @connection ||= Faraday.new(config.base_path, headers: BASE_REQUEST_HEADERS, request: request_options) do |conn|
+        conn.use :breakers
+        conn.adapter :httpclient
```

Comment:
It wasn't specifying an adapter before and needs to do so for this block configuration. I used httpclient because its what is being used by Sunsets, but I'll switch it back to the default one, as that's what's done in the secure messaging thing.